rules.md

```diff
@@ -425,274 +284,367 @@ The Meta % is an added commission that an Admin earns for processing a task clai
+The Proposal Owner (PO) is the [Admin](#4-admins) user who creates [Proposals](#64-proposals) to the Dash Network for and on behalf of the Incubator, and handles (but isn't liable for) custody and transactions of the Dash received on behalf of Admins.
 
-Admins must enter Approved Claims into the [Claim Awards List](https://docs.google.com/spreadsheets/d/1mhXlo4ylqWLLSYN4MGiLWlp7Gq3jrsDt0kB701dwMNU/edit#gid=0), filling in all columns and double checking the user’s payment address and reward amount, to enable the claim to be awarded.
+The PO agrees to use funds solely for the purposes of awarding [Task Claims](#26-claims) that were approved by Admins and are inline with the Rules in this document.
 
-Once an approved Claim is awarded (the txid was entered into the sheet), the Bounty Admin should reply to the user on the Bounty Card with the txid of the Dash transaction.
+The PO role is intended to be a temporary role, until the Incubator can be implemented as a fully decentralized dApp in future, with Proposal Dash awarded to a Contract on Dash Platform and distributed autonomously using code.
 
-## 4.4 Admin Rewards
+Before the full dApp implementation, and before fully decentralized mechanisms are innovated to ensure that all users are adhering to these Rules and operations of the App are as efficient as possible, the PO is currently granted some additional rights equivalent to a 'super-user' within the App as follows:
 
-Admins receive a % of the reward for Tasks on Bounty’s they administer, added to the Task reward, using the rates defined in the [Rewards List](#25-price-list).
+ - Appoint or transfer Primary or Secondary Admin positions on a Bounty
+ - Pause/Unpause a Bounty or some of the Tasks
+ - Appoint or remove Admins from the Incubator
+ - Create Tasks on bounties when Primary/Secondary aren't available
+ - Mediate updates to these Rules
+ - Any other actions to enforce adherence to these rules
 
-For example, if a Task Reward is 1 Dash and the Admin Reward is fixed at 10%, the user completing the Task is awarded the 1 Dash and the Bounty Admin is awarded an additional 0.1 Dash.
+ If the Proposal Owner needs to exercise any of these rights, they must leave a comment (e.g. on Trello or Github depending on the action) detailing the action and explaining the reasons as to why an action was taken.
 
-To receive rewards Admins must enter in the additional info in the Claim Awards List inline with each user’s Task claim, stating their name, Dash address and the reward amount (auto calculated).  The txid for the transaction should be pasted by the Admin who sends it.
+The Proposal Owner can be replaced through a vote where at least 51% of Admins vote for a new Admin to be the PO.  If that occurs, the newly elected PO will assume all PO responsibilities and the existing PO will transfer all funds from the Proposal Wallet to a wallet controlled by the new PO.
```

Comment:
again, "> 50%" might be more accurate and easy to reason about.

Reply:
done, thanks